README.rst

```diff
@@ -13,24 +13,32 @@
 .. image:: https://img.shields.io/pypi/v/openupgradelib.svg
    :target: https://pypi.python.org/pypi/openupgradelib
    :alt: Pypi Package
+   
+.. image:: https://img.shields.io/badge/License-AGPL%20v3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0
+   :alt: License: AGPL v3
```

Comment:
```suggestion
.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
   :target: https://www.gnu.org/licenses/agpl
   :alt: License: AGPL-3
```

Reply:
Lol, I literally cut and paste that from a recommended gist - no problems.

Reply:
I think I should revert the image, that png is dog ugly compared to the nice clean svg. Is there any reason we want png specifically?

Reply:
Oh, as you wish. Just FYI svg files are not rendered in apps, thus I always put .png in readme files by default.

Reply:
Good enough reason for me, although the other images are svg's, I'll leave it.

Reply:
AFAIK, SVG are rendered normally, and if not, we should fight/fix that. Where do you say it's not rendered?

Reply:
https://github.com/OCA/maintainer-tools/commit/52f41366ab39ea90e66f3e3a2b5cd7691b4219df

Reply:
Look the date of such change... I think this must be re-evaluated.

Reply:
Well in any case, this is not an app. 

Should I maybe convert to Markdown too? RST was a bit funky to edit remembering the slight syntax differences.

Reply:
Yeah, this is not an app. Let's put svg then!

Reply:
And we need to review the criteria for the rest, as Odoo has evolved. First of all, we nee to know where the incompatibility was.